pkg/infrastructure/data_index.go

```diff
@@ -30,6 +33,14 @@ const (
 	dataIndexHTTPRouteEnv = "KOGITO_DATAINDEX_HTTP_URL"
 	// Data index WS URL env
 	dataIndexWSRouteEnv = "KOGITO_DATAINDEX_WS_URL"
+	// Default Proto Buf file path
+	defaultProtobufMountPath = KogitoHomeDir + "/data/protobufs"
+	// Proto Buf folder env
+	protoBufKeyFolder string = "KOGITO_PROTOBUF_FOLDER"
+	// Proto Buf watch env
+	protoBufKeyWatch string = "KOGITO_PROTOBUF_WATCH"
+	// Proto Buf volume mode
+	protoBufConfigMapVolumeDefaultMode int32 = 420
```

Comment:
Can you move this constant to https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/framework/file_permissions.go ?

Reply:
done